utils.ts

```diff
@@ -90,6 +90,9 @@ export const convertQSPToGeocoderArgs = (
   }
   if (size) {
     geocoderArgs.size = size
+  } else {
+    // Safe, performant default
+    geocoderArgs.size = 4
   }
```

Comment:
This could be `geocoderArgs.size = size || 4` couldn't it?

Reply:
Although it doesn't keep with the structure of the above code, so I can see it both ways.

Reply:
This is a good call thank you! Slept right through this one trying to keep with the pattern